src/lib/config.rs

```diff
@@ -9,11 +9,13 @@ pub const SEXXI_GIT_DIR: &'static str = "$HOME/scratch/sexxi-rust/.git";
 pub const SEXXI_WORK_TREE: &'static str = "$HOME/scratch/sexxi-rust";
 pub const SEXXI_PROJECT: &'static str = "rust";
 
+// Default testing repo
+pub const SEXXI_TEST_PROJECT: &'static str = "sexxi-webhook-test";
+
 pub const SEXXI_REMOTE_HOST: &'static str = "sorbitol";
 pub const SEXXI_LOG_FILE_DIR: &'static str = "www/build-logs";
 
-// TODO(azhng): we need to template out the user name here.
-pub const BUILD_LOG_BASE_URL: &'static str = "https://csclub.uwaterloo.ca/~z577zhan/build-logs";
+pub const BUILD_LOG_BASE_URL: &'static str = "https://csclub.uwaterloo.ca/~rlmmfruy/build-logs";
```

Comment:
nit: can we just quickly refactor this into a function that returns the full URL using the $USER env var? otherwise everyone who checkout the code need to modify this string.

Reply:
Yes, my bad